index.html

```diff
@@ -146,21 +224,37 @@ <h4>Skills</h4>
             </ul>
             <img src="images/ic_skills.svg" alt="">
         </section>
+        
+        <div class="dots-teal">
+            <img src="images/image_dots_with_teal.png" alt="">
+        </div>
+
 
     </section>
 
-    <section class="form">
-        <img src="images/logo.svg" alt="">
+    <section class="form bckrg-form">
```

Comment:
- [ ] I noticed that the form section Is missing in the mobile version of your portfolio. Kindly fix this please.
### Your design
![1](https://user-images.githubusercontent.com/94551484/208278780-0bbab00f-1b0e-4350-aa8f-0145a29d83f7.png)
### Figma
![f1](https://user-images.githubusercontent.com/94551484/208278783-521393d7-4f8e-48b6-b0f5-3fec35857e04.png)